protractor/local.config.ts

```diff
@@ -1,10 +1,11 @@
 import { Config,browser } from 'protractor';
-
+import { reporter } from './helpers/reporter';
 export const config: Config = {
   framework: 'jasmine',
   specs: [ '../test/google.spec.js' ],
   seleniumAddress: 'http://localhost:4444/wd/hub',
   onPrepare: () => {
+    reporter();
```

Comment:
this should go outside of the onPrepare function

Reply:
![image](https://user-images.githubusercontent.com/33093117/88487284-49722100-cf49-11ea-9cf1-f80bc2e7142b.png)

Done

Reply:
The line "reporter()" if I put outside of the onPrepare function, it throws an error.  The guide says to put it into the onPrepare function
![image](https://user-images.githubusercontent.com/33093117/88487317-a40b7d00-cf49-11ea-9134-1491b203b28b.png)

Reply:
You're right, I'm sorry, got confused.